scripts/docs_pyauditor.sh

```diff
@@ -6,20 +6,27 @@ ENV_DIR=${ENV_DIR:=".env_test"}
 
 function make_docs() {
   cd pyauditor
-	python -m venv $ENV_DIR
-	source $ENV_DIR/bin/activate
-  pip install maturin
+  if [[ -z "${CI_MODE}" ]]
+  then
+    python -m venv $ENV_DIR
+    source $ENV_DIR/bin/activate
+    pip install --upgrade pip
+  fi
+  if [[ -z "${SKIP_COMPILATION}" ]]
+  then
+    pip install maturin
+    SQLX_OFFLINE=true maturin develop
+  fi
   pip install sphinx
   pip install sphinx_rtd_theme
   pip install myst-parser
-  SQLX_OFFLINE=true maturin develop
   cd docs
   make html
 }
 
 cleanup_exit() {
   setsid nohup bash -c "
-    rm -rf $ENV_DIR
+  rm -rf $ENV_DIR
```

Comment:
Should we only delete the ENV_DIR if it was also created with this sciprt? I.e. add a condition `if [[ -z "${CI_MODE}" ]]`?